packages/workflow/src/errors/node-api.error.ts

```diff
@@ -127,6 +128,10 @@ export class NodeApiError extends NodeError {
 	) {
 		super(node, errorResponse);
 
+		if (!httpCode && errorResponse instanceof AxiosError) {
+			httpCode = errorResponse.response?.status?.toString();
+		}
```

Comment:
The problem is here is the typing on `errorResponse` makes this very combersome to understand. maybe we can merge this change for now, and revisit refactoring the typings for `NodeApiError` / `NodeError`, so that we clearly distinguish between errors and json error responses.